contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -53,14 +33,23 @@ end
 @event
 func deposit_handled(l2_token : felt, account : felt, amount : Uint256):
 end
+
+@event
+func mint_rewards_initiated(l2_reward_token : felt, account : felt, amount : Uint256):
+end
+
+@storage_var
+func rewAAVE_token() -> (rewAAVE : felt):
+end
 # Constructor.
 
 # To finish the init you have to initialize the L2 token contract and the L1 bridge contract.
 @constructor
 func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
-        governor_address : felt):
+        governor_address : felt, rewAAVE : felt):
```

Comment:
I think initialising the rewAave token in the constructor will create a circular dependency because, we want the token_bridge to be the owner of the rewAave token and we won't have the rewAave token's address until we have the token_bridge's address

Reply:
I've seen that you've added a reward tokens setter on #6  so we will go with it